kodilla-stream/src/main/java/com/kodilla/stream/array/ArrayOperations.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.stream.array;
+
+import java.util.stream.IntStream;
+
+public interface ArrayOperations {
+
+    static double getAverage(int[] numbers) {
+        IntStream.range(0, numbers.length)
+              .map(n -> numbers[n])
+              .forEach(System.out::println);
+
+        return IntStream.range(0, numbers.length)
+              .map(n -> numbers[n])
+              .average().
+              getAsDouble();
```

Comment:
niebezpiecznie uzyc getAsDouble() - poszukaj, jak poprawnie to zrobic

Reply:
ok